packages/astro/src/content/types-generator.ts

```diff
@@ -391,6 +392,8 @@ async function writeContentFiles({
 			entries: {},
 		};
 	}
+
+	let contentCollectionsMap: CollectionEntryMap = {};
```

Comment:
There's nothing wrong for language tooling to also include data collections in the manifest (it just won't take them into account), but it caused crashes here because data collections are defined differently